### ISSUE TITLE
Pets attacking (maybe even killing!?) bot members of the crew

### ISSUE THREAD

**Issue by Benzin73** [Regalis11/Barotrauma#4889]
- I have searched the issue tracker and saw this issue reported as "question", but i see its been closed a while ago. And it is still happening. So i tought id report it again.

**Description**
NOTE: All of this happened in single player campaign.

The first pet i hatched a couple of weeks ago, was a Psilotoad. It was well fed and happy, but it soon started attacking two of my guard bots that i assigned as gunners. It dint attack anyone else. only those 2 guards, causing little (but constant) damage to them. I got rid of the pet by dumping it at the next station because i was running out of meds as the medic kept having to heal the wounds.
The pet had a nametag on it, and it started happening about 2 missions after i got it. It happend on a Dugong sub, 6 crew (me + 5 bots).

On my current run, i hatched another one, a Cthulhu. And on the very first mission since i got the pet, suddenly one of my crew just died. The sub was not under attack, there were no monsters on board, no fires, no lack of oxigen, nothing on the chat box, no cries for help. **his name bar just suddenly went red & dissapeared**. When i entered the room to see what had happened, the room was completely clear of any danger, but on the floor was the **decapitated** corpse of my engineer, and the cthulhu pet flying near it.
I did not actually see the pet attack the crew member, but there was nothing in the room that could have injured him (besides the pet), and the crewmember was healty before. The pet was not hostile to me when i entered the room.
This pet had no nametag, as i read that this issue seemed to happen only after giving them nametages. But it happened anyway.
This was on an Orca sub, me + 7 bots (8 total crew).

**Steps To Reproduce**
On a single player campaign, Get a few extra crew members, and hatch an egg to get either a Cthulhu or a Psilotoad. And just do a couple of missions. It has happened to me onboard Dugong and an Orca subs. One pet had a nametag, the other (the fatal incident) did not.
I dont know how likely it is to happen again. But for me it happened 2 times of the 2 times i got a pet (100%) of the time.

**Version**
Current stable 0.11.0.10 - Windows 10 64bits.

**Additional information**
Forum topic i started for this issue:
https://github.com/Regalis11/Barotrauma/discussions/4886

Steam forum topic where another user reported similar problems, and i reported the first incident
https://steamcommunity.com/app/602960/discussions/0/2999920878445738792/

**Comment by Regalis11** [comment]
Duplicate of https://github.com/Regalis11/Barotrauma/issues/4364, fix coming up in the next update
